vllm/config.py

```diff
@@ -99,6 +103,11 @@ def __init__(
         self.quantization_param_path = quantization_param_path
         self.enforce_eager = enforce_eager
         self.max_context_len_to_capture = max_context_len_to_capture
+        if self.max_context_len_to_capture is not None:
+            logger.warning("`max_context_len_to_capture` is deprecated. "
+                           "Use `max_seqlen_to_capture` instead.")
```

Comment:
I'm OK with having a breaking change here since most users do not adjust this parameter.